app/grandchallenge/archives/models.py

```diff
@@ -290,15 +291,19 @@ class ArchiveItem(CIVForObjectMixin, UUIDModel):
     values = models.ManyToManyField(
         ComponentInterfaceValue, blank=True, related_name="archive_items"
     )
+    title = models.CharField(max_length=255, default="", blank=True)
```

Comment:
I think theoretically all title related model definitions, the field, the constraint and the string method could be a mixin that you reuse for items and display sets. The only tricky part is getting the base object field name in the constraint definition. I think that would need to be done in the init method. Just a suggestion though

Reply:
This Needs a weird construct around the unique restraint, which makes it a lot less readable. Not worth the damage to readability imho.